src/snowflake/snowpark/functions.py

```diff
@@ -5774,6 +5774,9 @@ def object_pick(obj: ColumnOrName, key1: ColumnOrName, *keys: ColumnOrName) -> C
 def vector_cosine_distance(v1: ColumnOrName, v2: ColumnOrName) -> Column:
```

Comment:
Please also add a `@depricated` decorator here. How long will this be supported on the server side?
cc: @sfc-gh-yixie for API specific recommendation

Reply:
Added. We'll be removing support for this on the server side in the coming weeks.

Reply:
We haven't dealt with the server side removing a function before. Is it PrPr or PuPr?

Reply:
This function was PrPr: we went to PuPr (and now GA) without this function being supported server-side (it's only currently enabled for customers who were part of the PrPr).

Reply:
@sfc-gh-rpanchapakesan I don't see you added a `@depricated` decorator. Could you add it. Refer to https://github.com/snowflakedb/snowpark-python/blob/de15dd1426a4b436d1d8968186af914331aaf694/src/snowflake/snowpark/dataframe.py#L3410